cupyx/scipy/ndimage/measurements.py

```diff
@@ -425,6 +425,31 @@ def sum(input, labels=None, index=None):
     return _ndimage_sum_kernel(input, labels, index, index.size, out)
 
 
+def sum(input, labels=None, index=None):
```

Comment:
Maybe just a nitpick: Can we simply copy `sum_label`'s docstring? Something like
```python
sum.__doc__ = sum_labels.__doc__.replace('scipy.ndimage.sum', 'scipy.ndimage.sum_labels')
```

Reply:
There is a `Note:` section added here, though. The other option is to just omit the parameter and result descriptions and refer users to the docstring of `sum_labels` instead.

Reply:
Ohhh I overlooked! OK then it should be alright...